roles/sap_hana_install/defaults/main.yml

```diff
@@ -113,6 +113,10 @@ sap_hana_install_check_sidadm_user: true
 # an existing HANA system.
 sap_hana_install_new_system: true
 
+# Set to `true` to ensure the SAP system user '{{ sid | lower }}adm' is non-expiring.
```

Comment:
Please move this section to right above the section for `sap_hana_install_create_initial_tenant` so that it is in the same order as in the tasks file.